neural_compressor/torch/algorithms/weight_only/utility.py

```diff
@@ -441,3 +441,637 @@ def quant_weight_w_scale(weight, scale, zp, group_size=-1, dtype="int"):
             int_weight_tmp.add_(zp[:, -1].unsqueeze(1))
         int_weight[:, leng * group_size :].copy_(int_weight_tmp.round_())
     return int_weight
+
+
+# -------------- AWQ ---------------------------
```

Comment:
If you have any exposed functions or classes, please add them to `__all__` list.